system_tests/bigtable.py

```diff
@@ -93,3 +93,59 @@ def test_list_clusters(self):
             cluster_existence = (cluster in EXISTING_CLUSTERS or
                                  cluster == Config.CLUSTER)
             self.assertTrue(cluster_existence)
+
+    def test_reload(self):
+        # Use same arguments as Config.CLUSTER (created in `setUpModule`).
```

Comment:
Gotcher will do (holding off sending change until we resolve the other comment).